roles/maintenance_10_linux/tasks/main.yml

```diff
@@ -165,13 +168,14 @@
     regexp: "^root:"
     line: "root: {{ linux_serverlogs_root_alias }}"
     insertbefore: EOF
+  when: "'postfix' in ansible_facts.packages"
```

Comment:
Is /etc/aliases _only_ used by postfix in every distribution?

Reply:
No, it's used by sendmail and exim as well. However, our serverlogs setup uses postfix, and I'd argue that it thus doesn't make much sense to check for a serverlogs alias if postfix is absent.